composer.json

```diff
@@ -15,7 +15,7 @@
     }
   ],
   "require": {
-    "php": ">=5.6 || ^7.0",
+    "php": ">=5.6 || ^7.0 || ^8.0",
```

Comment:
Since you removed the testing for 5.6 might as well remove support for it. Also fix 7.0.. make sure it support 7.1 and up.